.gitignore

```diff
@@ -1,5 +1,6 @@
 *.o
 *.so
+*.bc
```

Comment:
Q: Why do we need that ?

Reply:
PG11 introduced JIT feature. We now have `.bc` files when building the project with LLVM. This change should  be done when we introduced PG11 support. Better late then never.